compiler/il/Aliases.cpp

```diff
@@ -183,6 +187,7 @@ OMR::SymbolReference::getUseonlyAliasesBV(TR::SymbolReferenceTable * symRefTab)
             case TR_transactionExit:
             case TR_newObject:
             case TR_newObjectNoZeroInit:
+            case TR_acmpHelper:
```

Comment:
can we make the names consistent between the symref and the helper - this really caused me to stumble in reading the code...

Reply:
Do you mean making the helper and non-helper names consistent? Like changing the name of the non-helper to be `acmpNonhelperSymbol`?

Reply:
Well it just seems funny to have the enum be TR_acmpHelper and the name of the symbol reference be `objectEqualityComparison` if there is a good reason for it that is fine, but difference for the sake of difference seems needlessly confusing...

Reply:
So, both `TR_acmpHelper` and `objectEqualityComparison` (which I am now realizing should be `objectEqualityComparisonSymbol`) are enum values. `objectEqualityComparison` is a non-helper introduced in this PR to represent object equality that is more complex than simple reference equality, as discussed above. `TR_acmpHelper` (introduced in #4841) is a VM helper needed by OpenJ9 that happens to be used to implement the functionality of `objectEqualityComparison`. In my mind, they are very different entities and, hence, have different names.

It's unfortunate that both have to exist in OMR but the current architecture means `TR_acmpHelper` has to be defined in OMR even though it's only going to be used in OpenJ9.

Reply:
Ok that makes sense - I withdraw my question/comment ... happy to proceed once the name matches the convention having a Symbol suffix.